docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -1261,17 +1257,67 @@ Execute the below command in the docker for the BLE+Thread pairing:
 |===
 
 
-==== Post-Test Steps
+===== Post-Test Steps
 
 Factory reset the DUT again → The test fills tons of stuff and the device will be in an odd state of ACL’s. This will be fixed once there is ample time to clean up after the test is completed by sending commands to, for example, remove the fabrics joined.
 
-==== Possible Issues
+===== Possible Issues
 
 * Failing at Step 9 during execution of TC_RR_1_1:
 .. Some DUT’s have an incorrectly-configured UserLabel cluster where the backend is not implemented due to SDK example issues where some examples have the backend and others do not. This will fail at the last step (“Step 9: Fill UserLabel clusters on each endpoint”), with FAILURE writes. To override the test not to run this step, you can add “*--bool-arg skip_user_label_cluster_steps:true*“ to the command line of *TC_RR_1_1.py*, at the end.
 .. Not having the *$PATH_TO_PAA_ROOTS* set properly when starting the docker or not having PAA roots certificates at that path.
 .. Follow the instructions for item 2 in <<prerequisite, Section 8.5.1, Prerequisite>>.
 
+==== Run Tests on the TH User Interface
+
+Some automated Python scripts are available in TH User Interface.
+
+To execute the tests, the parameters *discriminator*, *setup_code* and *pairing_mode* need to be filled in the device configuration parameters (*dut_config*).
+
+To configure specific/custom parameters, please edit the project configuration to include the parameters in the session (*test_parameters*).
+
+Project configuration example:
+[source,xml,options="unbreakable"]
+----
+{
+  ...
+  "dut_config": {
+    "discriminator": "3840",
+    "setup_code": "20202021",
+    "pairing_mode": "onnetwork",
+    "chip_tool_timeout": null,
+    "chip_tool_use_paa_certs": false
+  },
+  "test_parameters": {
+    "paa-trust-store-path": "/credentials/development/paa_roots"
+    "storage-path": "admin_storage.json"
+  }
+  ...
+}
+----
+
+Considering this project configuration and selecting tests TC_ACE_1_1 and TC_ACE_1_2. 
+
+image:images/img_61.png[]
+
+This configuration will result in the following execution steps:
+. Perform DUT commissioning
+. Run the TC_ACE_1_2 test (without commissioning)
+. Run the TC_ACE_1_3 test (without commissioning)
```

Comment:
This is not being shown as a list

<img width="1073" alt="Screenshot 2023-12-14 at 15 54 19" src="https://github.com/project-chip/certification-tool/assets/116589288/e1546bd9-f53f-4eb0-a506-2e0e53f3fdd7">

Reply:
Done :)